### DISCUSSION TITLE
chat between two people

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#9342] [Help]
Hello everyone, has anyone created a chat between two people? I'm specifically having trouble with the layout. Maybe someone has some tips or can point me in the right direction. I'm not very experienced with frontend development. When using ready-made components with Tailwind, the entire layout falls apart.

**Comment by Saifallak** [comment]
any luck with the ui?

**Reply by irfanandratama** [reply]
It would be nice if someone manage to do it

**Reply by Saifallak** [reply]
https://github.com/199ocero/filachat
